internal/transform/scope-css_test.go

```diff
@@ -0,0 +1,199 @@
+package transform
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	tycho "github.com/snowpackjs/astro/internal"
+	"github.com/snowpackjs/astro/internal/test_utils"
+)
+
+func TestScopeStyle(t *testing.T) {
+	// note: the tests have hashes inlined because it’s easier to read
+	// note: this must be valid CSS, hence the empty "{}"
+	tests := []struct {
+		name   string
+		source string
+		want   string
+	}{
+		{
+			name:   "class",
+			source: ".class{}",
+			want:   ".class.astro-XXXXXX{}",
```

Comment:
@jonathantneal No pressure at all, but if you have a moment could you review some of these examples / suggest some of your own? The `source` is the written code, and the `want` is the expected output.